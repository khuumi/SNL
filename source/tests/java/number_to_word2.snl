start main:
    numf is 15.2323
    do show to "this should print " + (do number_to_word to numf) + " to console."
done
