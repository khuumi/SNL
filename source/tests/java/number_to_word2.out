this should print 15.2323 to console.
